UICatalog/Scenarios/Shortcuts.cs

```diff
@@ -320,9 +320,7 @@ private void App_Loaded (object sender, EventArgs e)
 
         var bgColor = new ColorPicker ()
         {
-            CanFocus = false,
-            BoxHeight = 1,
-            BoxWidth = 1,
+            CanFocus = false, Width = 17
```

Comment:
`CanFocus` of false is set on all controls so I have left in... don't really understand this Scenario though.

Reply:
`Shortcut`'s `CommandView` can be any view. In most cases, it does not make sense for that view to be focusable (e.g. when `Shortcut` is used w/in `StatusBar`. 

I was using this to test all that.

Reply:
It's important that Views (e.g. ColorPicker) support mouse events even if `!CanFocus || !HasFocus`.